### PR TITLE
add comment about text field default

### PR DESCRIPTION
**Pull request by dcousens**
A question was asked why the `text` field was `isNullable` where others use `getResolvedIsNullable`.
My suspicion is that Keystone 6 has prioritized a zero-length string as a reasonable default instead as it is additionally falsy,  compared to the null?
If no other reasoning present's itself, I'm happy to make that canon.

### CODE REVIEW

File: packages/keystone/src/fields/types/text/index.ts [hunk 1]
```diff
@@ -70,6 +70,7 @@ export const text =
       },
     };
 
+    // defaulted to false as a zero length string is preferred
```
Comment (dcousens) [inline packages/keystone/src/fields/types/text/index.ts:73]:
```suggestion
    // defaulted to false as a zero length string is preferred to null
```

Comment (bladey) [reply packages/keystone/src/fields/types/text/index.ts:73]: // defaulted to false as a zero length string is preferred to null
```suggestion
    // defaulted to false as a zero length string is preferred
```

**Review by bladey — APPROVED** [review]

File: packages/keystone/src/fields/types/text/index.ts [hunk 2]
```diff
@@ -70,6 +70,7 @@ export const text =
       },
     };
 
+    // defaulted to false as a zero length string is preferred to null
```
Comment (bladey) [inline packages/keystone/src/fields/types/text/index.ts:73]:
```suggestion
    // defaulted to false as a zero length string is preferred to null
```

**Review by bladey — CHANGES_REQUESTED** [review]

**Review by bladey — APPROVED** [review]

**Comment by molomby** [comment]
There's now a discussion on [text field nullability defaults](https://github.com/keystonejs/keystone/discussions/7158) that relates to this.

**Pull request merged**